io/src/test/scala/fs2/io/file/BaseFileSpec.scala

```diff
@@ -25,7 +26,12 @@ class BaseFileSpec extends Fs2Spec {
     Stream.eval(IO(Files.write(file, Array[Byte](0, 1, 2, 3))).void)
 
   protected def modifyLater(file: Path): Stream[IO, Unit] =
-    Stream.eval(IO(Future(List[Byte](0, 1, 2, 3).foreach(waitAndAppend(file)))).void)
+    Stream
+      .range(0, 4)
+      .map(_.toByte)
+      .covary[IO]
+      .metered(250.millis)
+      .evalMap(b => IO(Files.write(file, Array(b), StandardOpenOption.APPEND)).void)
```

Comment:
Instead of using `Files.write` here, can we use `fs2.io.file.writeAll`?

Reply:
Well, since `fs2.io.file.writeAll` is tested in the same file, I don't feel confortable using it here.

Otherwise, if ever `file.writeAll` breaks, even if `file.keepReading` is alright, neither test will pass...

What do you think?

Reply:
I think that's okay and should be protected by unit tests for `writeAll`.

Reply:
Okay... So I moved the test _after_ that of of `writeAll`.